lib/linebuffer.js

```diff
@@ -45,121 +45,123 @@ const needsRetry = (error, response, body) => {
         httpErrorTime = null;
     }
 
+    buffer = [];
```

Comment:
Why this line is needed?

Reply:
to empty after each flush

Reply:
I think Wei is asking: doesn't line 70 do that already?

Reply:
yup - discussed already - forgot to update: remove that part